### DISCUSSION TITLE
Help on adding thebe interactivity to code-cells executed on build by MyST-nb.

### DISCUSSION THREAD

**Discussion by MantiMantilla** [executablebooks/meta#186] [Q&A] [answered]
I have a book that displays `code-cell`s from a MyST-NB .md and executes them on `sphinx-build`.  I also want to add interactivity to those cells with sphinx-thebe but I'm running into an issue wherein `code-cell`s get executed on build but thebe won't make them interactive when I click the button (I believe because thebe works on `code-block`s, not cells). If I change my `code-cell`s to `code-block`s, thebe will still not recognize them when I press the button but they also won't execute on build. What am I doing wrong?

My `code-cell`s:

`````
```{code-cell} ipython3
---
class: thebe
tags: [enable-thebe]
---
import numpy as np
np.random.seed(1337)
data = np.random.randn(2, 100)
print(data[1, :10])
```

```{thebe-button} Correr el código
```
`````

This is how they render on the static site:

![image](https://user-images.githubusercontent.com/28278569/100377506-5ed28700-2fdf-11eb-8413-49cd57d45f76.png)

Making the cells, blocks only removes the output. Thebe has no issue launching the mybinder server. I just can't get it to recognize the input and output cells. What should I do?

A link to the repo: 

[Theory of Computation - Criptography](https://github.com/MantiMantilla/Theory-of-Computation-Encryption)

pd. I've used Jupyter Book. It's wonderful! But I really want to use custom themes.

**Comment by MantiMantilla** [comment, marked as answer]
Nevermind the issue. I was just being silly.
Because I'm using the _Furo_ theme, I had to tell _thebe_ specifically the class of div with which to identify the inputs and outputs (I had typed in the wrong classes).
I don't see how I can delete the post since this information is literally in the docs and so I'll show explicitly the config I used for this theme.
```
thebe_config = {
    "repository_url": "https://github.com/MantiMantilla/Theory-of-Computation-Encryption",
    "path_to_docs": "criptografia-book",
    "repository_branch": "main",
    "selector": "div.section",
    "selector_input": "div.cell_input",
    "selector_output": "div.cell_output",
    #"codemirror-theme": "blackboard"  # Doesn't currently work
}
```

**Reply by chrisjsewell** [reply]
Alway good when someone answers their own question thanks 😆
